### ISSUE TITLE
AtlasMessagesAdapter uses mRecyclerView.getHeight() to determine height of ThreePartImageCells, which changes if the keyboard is visible

### ISSUE THREAD

**Issue by elliott-accolade** [layerhq/Atlas-Android#135]
Device: Moto X <6.0.1>

Noticed that inside the `AtlasMessagesAdapter`, the `bindCellViewHolder` method uses the height of the `RecyclerView` to determine the size of the ThreePartImage Cells. The code in question is here:
```
        // TODO: subtract spacing rather than multiply by 0.8 to handle screen sizes more cleanly
        int maxHeight = (int) Math.round(0.8 * mRecyclerView.getHeight());
```

The problem with this is that the `RecyclerView` height changes depending on whether the keyboard is visible or not, so if I take a picture with the `CameraSender`, then send a text message, the `CellHolder` is resized and displays the image preview, shrunken.

**Comment by thecombatwombat** [comment]
@elliott-accolade We have a fix for the issue in #65, so I'm going to close this ticket. 

The fix should be released soon. Please let us know if you if you still have problems.

**Comment by elliott-accolade** [comment]
Thanks @thecombatwombat!
